tests/libnet/ipaddress.go

```diff
@@ -0,0 +1,29 @@
+package libnet
+
+import (
+	v1 "k8s.io/api/core/v1"
+	netutils "k8s.io/utils/net"
+)
+
+func GetIp(ips []string, ipv6 bool) string {
+	for _, ip := range ips {
+		if netutils.IsIPv6String(ip) {
+			if ipv6 {
+				return ip
+			}
+		} else {
+			if !ipv6 {
+				return ip
+			}
+		}
+	}
+	return ""
+}
```

Comment:
Could we have 2 functions `GetIPv4Addr` and `GetIPv6Addr` instead of relying on the boolean ?

Reply:
It means extra code on the calling method deciding which method to invoke. Can you please provide more argument why you think the two methods approach is better?

Reply:
Alona, Miguel, Or
please consider 
https://github.com/kubevirt/kubevirt/pull/4156#discussion_r492063352

this will remove the need of functions almost totally,
its just other appraoch, we test primary and secondary, we dont check ipv4 and ipv6, 
just the few cases that will do care which family it is will handle them in the test itself.

Reply:
Don't you need that code to compute the boolean on the caller never the less ? 

About the boolean: 
  - https://medium.com/@amlcurran/clean-code-the-curse-of-a-boolean-parameter-c237a830b7a3
  - https://www.informit.com/articles/article.aspx?p=1392524

Reply:
Passing a boolean implies the function does two things, which it indeed softly does.
As the caller already knows what is needed, passing a boolean adds another condition and therefore prone to bugs and limited if there is a need to extend something.

There is no need for extra code at the caller, the input should not be a boolean (to the table argument) but these functions... then you just call them, no questions needed.